src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -15,25 +15,25 @@ description: "Learn about the general Session Replay configuration fields."
 
 The following options can be configured on the root level of your browser-based Sentry SDK, in `init({})`:
 
```

Comment:
On all the tables (on this page and on the Privacy page), should we be using PlatformIdentifier components to ensure correct casing with in the tables?

Reply:
Even though this is top-level menu, it's only JS-configuration. Even if/when we expand to other platforms, we wouldn't be opt'ing in this page to those platforms, as too much of it wouldn't apply, including most options.

For that reason I believe this is fine as-is